privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -13,20 +14,30 @@
 from art.estimators.classification import TensorFlowV2Classifier
 import string
 
+# count of shadow training sets, must be even
+AMOUNT_SETS = 2
```

Comment:
I think this is ok right now but when we really implement a multi-class attack we have to edit this variable

Reply:
Changed in new pull request so I resolved the conversation.